interface/wx/thread.h

```diff
@@ -503,10 +503,25 @@ class wxThreadHelper
 
         @since 2.9.2
 
-        @see OnDelete()
+        @see OnDelete(), OnExit()
     */
     virtual void OnKill();
 
+    /**
+        Callback called by Exit() before actually exiting the thread.
+        This function will not be called if the thread was @ref Kill() killed.
+
+        This function can be overridden by the derived class to perform some
+        specific task when the thread is exited. Notice that it will be
+        executed in the context of the thread that called Exit() and <b>not</b>
+        in this thread's context. 
+
+        @since 2.9.2 - private, 3.1.3 - protected
```

Comment:
I'm not sure if this is going to come out correctly, so I'd rather just write

    Note that this function is protected since wxWidgets 3.1.1, but previously existed as a private method since 2.9.2.